src/vcpkg/portlint.cpp

```diff
@@ -24,24 +22,56 @@ namespace vcpkg::Lint
         return original_scheme;
     }
 
-    Status check_usage_forgot_to_install(Filesystem& fs, const SourceControlFileAndLocation& scf, Fix fix) {
-        auto portfile_path =  scf.source_location / "portfile.cmake";
+    namespace
+    {
+        static inline size_t find_ignore_case(const std::string& content, const std::string& find, std::size_t offset = 0)
+        {
+            auto it = std::search(
+                content.begin() + offset,
+                content.end(),
+                find.begin(),
+                find.end(),
+                [](unsigned char ch1, unsigned char ch2) { return std::tolower(ch1) == std::tolower(ch2); });
+            if (it != content.end()) return it - content.begin();
+            return std::string::npos;
+        }
+
+        static bool check_has_install_usage(const std::string& file)
+        {
+            size_t pos = 0;
+            while ((pos = find_ignore_case(file, "file(", pos)) != std::string::npos)
+            {
+                pos += 5;
+                auto cmd_end = find_ignore_case(file, ")", pos);
+                if (cmd_end == std::string::npos) return false;
+                auto usage = find_ignore_case(file, "usage", pos);
```

Comment:
```suggestion
                auto usage = file.find("usage", pos);
```
The filename will always be lowercase because on linux we have case sensitive file systems